# Bepaling van soortelijke warmte van water


# Introductie

Water heeft een enorme buffercapaciteit: je moet veel energie toevoegen om de temperatuur van water een graad te verwarmen. In dit practicum gaan we de soortelijke warmte van water bepalen door een bekende hoeveelheid water te verwarmen met een bekende hoeveelheid energie, en de temperatuurstijging te meten.

# Theorie
Omdat water een incompressibel media is, geld voor de soortelijk warmte $c$ van water dat $c=c_v=c_p$. Verder geld, als $v$ constant is, $c_v=\frac{du}{dT}$. Deze uitdrukking herleiden geeft:
\begin{align*}
du &=c_v \cdot dT \\
\int\, du &= \int c_v\, dT \\
\Delta u &= c_v \cdot \int\, dT \\
m \cdot \Delta u &=  m\cdot c_v\cdot \Delta T \\
Q &= m\cdot c_v\cdot \Delta T \\
c &= c_v = \frac{Q}{m\cdot\Delta T}
\end{align*}
hierin is:
- $Q$: warmte-energie in joule (J)
- $m$: massa in kilogram (kg)
- $c$: soortelijke warmte in joule per kilogram per kelvin (J/(kg·K))
- $\Delta T$: temperatuurverandering in kelvin (K) of graden Celsius (°C)

Bij de derde stap van het herleiden is aangenomen dat $c_v$ constant is bij verandering van temperatuur. In werkelijkheid zal deze licht veranderen als de temperatuur toeneemt maar deze verandering is bij kleine temperatuurs verschillen verwaarloosbaar klein.

Door gebruik te maken van een elektrisch verwarmings element kan de warmte $Q$ voor een bepaald tijdsinterval bepaalt worden. Voor het vermogen van het vermwarmings element geld namelijk $P= U \cdot I$, hierin is $P$ het vermogen in Watt(J/s) , $U$ de spanning in volt(V) en $I$ de stroomsterkte in ampere(A). De toegevoerde energie wordt dan weergeven door $Q=P\cdot t = U\cdot I \cdot t $, hierin is $t$ de tijd in seconden(s). De uiteindelijke uitdrukking voor $c$ die je dan krijgt is:
$$c = \frac{U\cdot I \cdot t}{m\cdot\Delta T}$$

# Methode en materialen

## Ontwerp
Een waterbad met bekende massa aan water wordt verwarmd met een elektrisch verwarmingselement dat een bekende hoeveelheid energie levert.
De temperatuur van het water wordt gemeten met een temperatuursensor.
Door de temperatuurstijging als functie van de tijd te meten kan de soortelijke warmte van water worden berekend.


## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Maatbeker
- Weegschaal
- Water
- Elektrisch verwarmingselement ($10 \mathrm{\Omega}$, $10 \mathrm{W}$)
- Voedingsbron 
- Thermometer of temperatuursensor
- Stopwatch of timer

```{figure} figures/cwater_setup.jpg
:width: 70%

Een schematische weergave van de opstelling
```

## Procedure
```{exercise}
1. Beschrijf de procedure die nodig is om de soortelijke warmte van water te bepalen.
2. Geef jouw beschrijving van de procedure aan een ander team, zij verzamelen voor jou de data op basis van de gegevens.
3. Voer het experiment van een ander team uit op basis van hun beschrijving.
4. Analyseer de data die het andere groepje voor jou heeft verzameld.
```


## Veiligheid
We maken gebruik van een $10 \mathrm{\Omega}$, $10 \mathrm{W}$ weerstand.
Deze wordt snel heet.
De bronspanning mag dan ook alleen aan wanneer de weerstand in het water zit.
Raak de weerstand niet aan tijdens het experiment.
Omdat de weerstand in het water zit, kunnen we wel het elektrisch vermogen hoger zetten zonder dat de weerstand oververhit raakt.
Het maximaal vermogen mag $40 \mathrm{W}$ zijn.
Daarbij moet de roerder wel aanstaan om de warmte goed te verdelen.

## Data analyse
De data van de metingen wordt gefit op de omgeschreven uitdrukking voor c.

# Resultaten


In [ ]:
import numpy as np
volt=11.9                       #voltage van voeding
amp=1.24                        #amperage van de voeding
mlwater=130                     #ml
rho_water=0.99937               #kg/L #de dichtheid wat water bij 16.8 celcius
m_water=mlwater*1e-3*rho_water  #kg, massa van het water

#metingen
temp = np.array([16.8, 16.9, 17.0, 17.1, 17.3, 17.4, 17.5, 17.6, 17.7, 17.8, 17.9,
       18.0, 18.2, 18.3, 18.4, 18.5, 18.6, 18.7, 18.8, 18.9, 19.1, 19.2,
       19.3, 19.4, 19.5, 21.1, 21.2, 21.3, 21.4, 21.5, 21.5, 21.6, 21.7,
       21.8, 21.9, 22.0, 22.6, 22.7, 22.7, 22.8, 22.9, 23.0, 23.0, 23.1,
       23.2, 23.3, 23.3, 23.4, 23.5, 23.6, 23.9, 24.0, 24.2, 24.3, 24.4])+273.15 #K

tijden= np.array([0,   5,  10,  15,  20,  25,   30  ,35,  40,  45,  50,  55,  60,  65,  70,  75,  80,  85,
        90,  95, 100, 105, 110, 115, 120, 125, 130, 135, 140, 145, 150, 155, 160, 165, 170, 175,
        180, 185, 190, 195, 200, 205, 210, 215, 220, 225, 230, 235, 240, 245, 250, 255, 260, 265,
        270]) #s


In [41]:
from scipy.optimize import curve_fit
#functie om mee te fitten
def T_func(t,c):
    '''De omsgeschrevn vorm van de uitdrukking voor c zodat er gefit kan worden'''
    T_0=min(temp)#de begintemperatuur
    return T_0+volt*amp*t/(c*m_water)

#de fit en resultaat
val,cov=curve_fit(T_func,tijden,temp)
print("De soortelijke warmte is bepaalt op %.0f +/- %.0e J/kg/K" %(val[0],np.sqrt(cov[0,0])))



De soortelijke warmte is bepaalt op 3930 +/- 5e+01 J/kg/K


# Discussie en conclusie

## Discussie
Uit deze proef volgt een waarde voor de soortelijkewarmte van water van $3930 +/-  5\cdot10^1$. Deze waarde wijkt af van de werkelijkewaarde. De belangrijkste oorzaken hiervoor zijn dat de massa van het water niet precies is bepaald, dat er kraanwater is gebruikt en geen gedistileerd water en dat er warmte uit het water is gestroomd waardoor de temperatuurmeting foutief is. 